Thanks for the PR! LGTM.

Looks like there is still an issue in the Python test, could you have a look?

```
2022-09-28T16:42:47.5708552Z =================================== FAILURES ===================================
2022-09-28T16:42:47.5708850Z __________________________ TestRelation.test_explain ___________________________
2022-09-28T16:42:47.5709114Z 
2022-09-28T16:42:47.5709271Z self = <test_relation.TestRelation object at 0x7f1c36b19610>
2022-09-28T16:42:47.5709612Z duckdb_cursor = <duckdb.DuckDBPyConnection object at 0x7f1c3da1db30>
2022-09-28T16:42:47.5709819Z 
2022-09-28T16:42:47.5709941Z     def test_explain(self, duckdb_cursor):
2022-09-28T16:42:47.5710196Z         con = duckdb.connect()
2022-09-28T16:42:47.5713968Z         con.execute("Create table t1 (i integer)")
2022-09-28T16:42:47.5714284Z         con.execute("Create table t2 (j integer)")
2022-09-28T16:42:47.5809127Z         rel1 = con.table('t1')
2022-09-28T16:42:47.5809757Z         rel2 = con.table('t2')
2022-09-28T16:42:47.5810121Z         join = rel1.join(rel2, 'i=j', 'inner').aggregate('count()')
2022-09-28T16:42:47.5810659Z >       assert join.explain() == 'Aggregate [count_star()]\n  Join INNER (i) = (j)\n    Scan Table [t1]\n    Scan Table [t2]'
2022-09-28T16:42:47.5811193Z E       AssertionError: assert 'Aggregate [c...an Table [t2]' == 'Aggregate [c...an Table [t2]'
2022-09-28T16:42:47.5811494Z E           Aggregate [count_star()]
2022-09-28T16:42:47.5811789Z E         -   Join INNER (i) = (j)
2022-09-28T16:42:47.5812062Z E         ?                -   -
2022-09-28T16:42:47.5812278Z E         +   Join INNER (i = j)
2022-09-28T16:42:47.5812484Z E               Scan Table [t1]
2022-09-28T16:42:47.5812708Z E               Scan Table [t2]
2022-09-28T16:42:47.5812843Z 
```

--- a/tools/pythonpkg/tests/fast/test_relation.py
+++ b/tools/pythonpkg/tests/fast/test_relation.py
@@ -198,7 +198,7 @@ class TestRelation(object):
         rel1 = con.table('t1')
         rel2 = con.table('t2')
         join = rel1.join(rel2, 'i=j', 'inner').aggregate('count()')
-        assert join.explain() == 'Aggregate [count_star()]\n  Join INNER (i) = (j)\n    Scan Table [t1]\n    Scan Table [t2]'
+        assert join.explain() == 'Aggregate [count_star()]\n  Join INNER (i = j)\n    Scan Table [t1]\n    Scan Table [t2]'
 
     def test_fetchnumpy(self, duckdb_cursor):
         start, stop = -1000, 2000

